site/docs/v1/tech/migration-guide/_slow-migration-timeline.adoc

```diff
@@ -0,0 +1,19 @@
+You can calculate a timeline by knowing the following:
+
+* How frequently the average user logs in
```

Comment:
nit: these sentences "dangle"

Reply:
Not sure how to fix these. I guess I could flip them around ("login frequency") but that doesn't work well with all of them.